bcda/logging/middleware.go

```diff
@@ -60,6 +60,8 @@ func (l *StructuredLogger) NewLogEntry(r *http.Request) middleware.LogEntry {
 		logFields["cms_id"] = ad.CMSID
 	}
 
+	logFields["transaction_id"] = r.Context().Value(CtxTransactionKey).(string)
```

Comment:
Do we need to check return values here, similar to the if statement above?

```go
if tid, ok := r.Context().Value(CtxTransactionKey).(string); ok {
  ...
}
```

Reply:
good catch, that is how we should be checking! I'll update that.